pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/ZKMetadataStore.java

```diff
@@ -120,7 +121,12 @@ public CompletableFuture<Optional<GetResult>> get(String path) {
                             return null;
                         });
                         future.complete(Optional.empty());
-                    } else {
+                    } else if (code == Code.CONNECTIONLOSS) {
+                        // There is the chance that we caused a connection reset by sending or requesting a batch
```

Comment:
Do we need to log something before retrying?

Reply:
Good idea